src/gtk/widgets/recent_info.rs

```diff
@@ -141,23 +142,22 @@ impl RecentInfo {
 
     pub fn has_group(&self, group_name: &str) -> bool {
         unsafe {
-            to_bool(
-                ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
-                                               group_name.borrow_to_glib().0))
+            to_bool(ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
+                group_name.borrow_to_glib().0))
         }
```

Comment:
Matching parentheses? What if I want to see what goes where without moving the cursor?

These changes introduce pointless code churn that has little to no effect on the number of lines. Talk about decorative purposes.